pkg/secret/encryption_other.go

```diff
@@ -99,6 +100,7 @@ func Decrypt(username, encrypted string, salt, nonce []byte) (string, error) {
 	if len(nonce) != NonceLength {
 		return "", fmt.Errorf(errors.IncorrectNonceLengthErrorMsg)
 	}
+	log.CliLogger.Debugf("Decrypting secret: %s", cipherText)
```

Comment:
only added debugf for decrypting not encrypting as I don't want to print plaintext secrets in logs.